src/platforms/common/data-management/event-grouping/server-side-fingerprinting.mdx

```diff
@@ -246,3 +246,12 @@ message:"connection error*" -> connection-error, {{ tags.server_name }}
 ```
 
 </DefinitionList>
+
+## Custom Titles
+
+When you use fingerprinting to group events together it can sometimes be useful to also change the default title of the event.  Normally the title of the event is the exception type and value (or the top most function names for certain platforms).  When you group by custom rules this title can often be misleading.  For instance if you group a logger together you might want to name the group after that logger.  This can be accomplished by setting the `title` attribute like this:
+
+```discover {tabTitle:Fingerprinting Config}
```

Comment:
what is discover? afaik thats not a highlighting dialect?

Reply:
@dcramer missed this comment but it's a custom highlighting dialect we added for the discover/search query syntax which is also the same syntax that fingerprinting rules use.